src/main/java/com/pinterest/secor/parser/TimestampedMessageParser.java

```diff
@@ -16,6 +16,8 @@
  */
 package com.pinterest.secor.parser;
 
+import com.google.protobuf.Timestamp;
```

Comment:
Timestamp is a protobuf specific class, should the new method 'toMillis' live in the  ProtobufMessageParser class?